BUILD

```diff
@@ -1,241 +1,280 @@
+    "gcr.io/{PROJECT_ID}/static-debian10:{COMMIT_SHA}": "//base:static_root_amd64_debian10",
+})
+
+STATIC.update({
+    "gcr.io/{PROJECT_ID}/static:" + label + "-" + arch: "//base:static_" + user + "_" + arch + "_debian9"
+    for arch in ARCHITECTURES
+    for (label, user) in LABEL_USERS
+})
+
+STATIC.update({
+    "gcr.io/{PROJECT_ID}/static-" + distro + ":" + label + "-" + arch: "//base:static_" + user + "_" + arch + "_" + distro
+    for arch in ARCHITECTURES
+    for (label, user) in LABEL_USERS
+    for distro in DISTROS
+})
+
+BASE = {
+    "gcr.io/{PROJECT_ID}/base:{COMMIT_SHA}": "//base:base_root_amd64_debian9",
+    "gcr.io/{PROJECT_ID}/base-debian9:{COMMIT_SHA}": "//base:base_root_amd64_debian9",
+    "gcr.io/{PROJECT_ID}/base-debian10:{COMMIT_SHA}": "//base:base_root_amd64_debian10",
+}
+
+BASE.update({
+    "gcr.io/{PROJECT_ID}/base:" + tag_base + "-" + arch: "//base:" + label + "_" + user + "_" + arch + "_debian9"
+    for arch in ARCHITECTURES
+    for (tag_base, label, user) in [
+        ("latest", "base", "root"),
+        ("nonroot", "base", "nonroot"),
+        ("debug", "debug", "root"),
+        ("debug-nonroot", "debug", "nonroot"),
+    ]
+})
+
+BASE.update({
+    "gcr.io/{PROJECT_ID}/base-" + distro + ":" + tag_base + "-" + arch: "//base:" + label + "_" + user + "_" + arch + "_" + distro
+    for arch in ARCHITECTURES
+    for (tag_base, label, user) in [
+        ("latest", "base", "root"),
+        ("nonroot", "base", "nonroot"),
+        ("debug", "debug", "root"),
+        ("debug-nonroot", "debug", "nonroot"),
+    ]
+    for distro in DISTROS
+})
+
+CC = {
+    "gcr.io/{PROJECT_ID}/cc:{COMMIT_SHA}": "//cc:cc_root_amd64_debian9",
+    "gcr.io/{PROJECT_ID}/cc-debian9:{COMMIT_SHA}": "//cc:cc_root_amd64_debian9",
+    "gcr.io/{PROJECT_ID}/cc-debian10:{COMMIT_SHA}": "//cc:cc_root_amd64_debian10",
+}
+
+CC.update({
+    "gcr.io/{PROJECT_ID}/cc:" + tag_base + "-" + arch: "//cc:" + label + "_" + user + "_" + arch + "_debian9"
+    for arch in ARCHITECTURES
+    for (tag_base, label, user) in [
+        ("latest", "cc", "root"),
+        ("nonroot", "cc", "nonroot"),
+        ("debug", "debug", "root"),
+        ("debug-nonroot", "debug", "nonroot"),
+    ]
+})
+
+CC.update({
+    "gcr.io/{PROJECT_ID}/cc-" + distro + ":" + tag_base + "-" + arch: "//cc:" + label + "_" + user + "_" + arch + "_" + distro
+    for arch in ARCHITECTURES
+    for (tag_base, label, user) in [
+        ("latest", "cc", "root"),
+        ("nonroot", "cc", "nonroot"),
+        ("debug", "debug", "root"),
+        ("debug-nonroot", "debug", "nonroot"),
+    ]
+    for distro in DISTROS
+})
+
+PYTHON3 = {
+    "gcr.io/{PROJECT_ID}/python3:{COMMIT_SHA}": "//experimental/python3:python3_root_amd64_debian9",
+    "gcr.io/{PROJECT_ID}/python3-debian9:{COMMIT_SHA}": "//experimental/python3:python3_root_amd64_debian9",
+    "gcr.io/{PROJECT_ID}/python3-debian10:{COMMIT_SHA}": "//experimental/python3:python3_root_amd64_debian10",
```

Comment:
Let's not add these. The current Python images don't have these tags. I actually plan to remove `{COMMIT_SHA}` from other images as well.

Reply:
Remove the COMMIT_SHA images from both pythons